content/docs/0.19.x/operate/backup_and_restore/index.md

```diff
@@ -137,3 +224,23 @@ kubectl apply -f $PROJECT_NAME-credentials.yaml
 ```
 
 After executing these steps, your projects and all events should be visible in the Keptn Bridge again.
+
+## Restore Configuration Service data to Keptn using Resource Service
+
+This section describes how to restore an acient data backup of the Configuration Service and apply it to a Keptn instance running a Resource Service. This can be applied only to projects which do not have a remote git upstream repository configured. The pre-requisities for these steps is to have the MongoDB data restored already.
```

Comment:
```suggestion
This section describes how to restore an ancient data backup of the Configuration Service and apply it to a Keptn instance running a Resource Service. This can be applied only to projects which do not have a remote git upstream repository configured. The pre-requisites for these steps is to have the MongoDB data restored already.
```
I am guessing that "acient" was a typo for "ancient".  Please correct if I am wrong.

How is it possible to have a project that does not have a remote git upstream repository with the current releases?  Maybe this is intended for people who are backing up an old release of Keptn and not moving to a current release that requires the upstream?  If so, may want to specify that.

Reply:
exactly, it is for people using s very old version of Keptn, who want to backup their data and move to the newest version (not upgrading, but completely new installation). In this case they should also be able to restore their data

Reply:
This also solves https://github.com/keptn/keptn.github.io/issues/1394